### DISCUSSION TITLE
Order in decreasing order with limit

### DISCUSSION THREAD

**Discussion by KrishGarg** [deta/cloud-docs#305] [Help] [answered]
Hi! I wanted to get data in decreasing order of a field but was limited to 30 records.
In SQL, it would be kinda like 
```sql
SELECT field FROM table 
ORDER BY field DESC
LIMIT 30
```

**Comment by abdelhai** [comment, marked as answer]
hi, limits are documented here: https://docs.deta.sh/docs/base/sdk#fetch
base doesn't support sorting by another column other than `key` (automatically sorted): https://github.com/orgs/deta/discussions/215#discussioncomment-2781638
